### ISSUE TITLE
Add aiida-sssp and/or aiida-pseudo to image

### ISSUE THREAD

**Issue by chrisjsewell** [marvel-nccr/ansible-role-aiida#35]
As discussed in #34

**Comment by sphuber** [comment]
Definitely wouldn't do both. If there is no rush, I would wait for `aiida-pseudo` to be published. It currently is working, see master branch: https://github.com/aiidateam/aiida-pseudo/commits/master
But I haven't released it yet, because I haven't had the time to discuss with other developers about the design/naming choices etc. Also some advanced functionality that was in `aiida-sssp`, i.e. to automatically get the cutoffs for a given structure, have not yet been ported.

**Comment by chrisjsewell** [comment]
Thanks @sphuber, so just to clarify `aiida-pseudo` will eventually subsume `aiida-sssp`?

**Comment by sphuber** [comment]
yes, `aiida-sssp=0.1.1` works for now, but only supports installing from Materials Cloud. The `aiida-pseudo` package will generalize the functionality to support generic pseudo families, not just SSSP and even not just UPF, and will allow to install from pseudos on local file system. They are incompatible in the sense that the entry points for the `Group` subclases are different.
